### PR TITLE
Fix pills so they allow for multiline answers

### PR DESCRIPTION
**Pull request by nishasy**
## Summary

The latest update to Wonder Blocks Pill give it a set height. This makes it cut off multi-line answers in the answer pills in the label-image widget.

To fix this, I added `height: "auto"` to the pill style to reset its height and stop the clipping.

Issue: LC-1664

## Test plan:

- Comment out the added `height: "auto"`
- Go to label-image widget storybook
  - http://localhost:6006/?path=/story/perseus-widgets-label-image--label-image-numberline
- Click on a number line dot and select any of the options.
- Confirm that the answer pill is cut off.
- Add back the `height: "auto"`
- Go back to the number line and confirm that the answer pill is fully displayed now.

### Before

Screenshot 2024-01-25 at 1 56 17 PM

### After

Screenshot 2024-01-25 at 1 56 31 PM

### CODE REVIEW

File: packages/perseus/src/widgets/label-image/answer-pill.tsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ export const AnswerPill = (props: {
                     style={[
                         style,
                         popperStyle,
-                        styles.relative,
```
Comment (nishasy) [inline packages/perseus/src/widgets/label-image/answer-pill.tsx:77]: I don't think `styles.relative` exists anymore?

Comment (nedredmond) [reply packages/perseus/src/widgets/label-image/answer-pill.tsx:77]: Thanks for getting rid of that!

**Comment by nedredmond** [comment]
Would you also please upgrade WonderBlocks as a part of this?

From root, you can run `yarn upgrade-interactive --latest` and choose all the `@khanacademy/wonder-blocks-` choices.

That way, we can confirm it looks right with the latest version of WB.

**Comment by nishasy** [comment]
@nedredmond Thanks, that might explain why I don't see the issue in the Perseus storybook!

**Review by nedredmond — APPROVED** [review]
Looks great-- just changeset patch `math-input` to fix the ❌ and you're good to land!

**Pull request merged**